mage_ai/frontend/utils/models/featureSet.ts

```diff
@@ -62,6 +62,33 @@ export function getFeatureSetStatistics(featureSet, featureUUID) {
   };
 }
 
+export function getFeatureSetInvalidValues(featureSet, featureUUID) {
```

Comment:
Don't we already get these stats in the `getFeatureSetStatistics` method? So calling this seems redundant. We could just add another property for `invalid_indices` since that one isn't included in the `getFeatureSetStatistics` method.

Reply:
This function does a different thing than the `getFeatureSetInvalidValuesAll`. Also this function is unused and is part of the next story to display in a chart what the values are.

Reply:
I'll merge both into the getFeatureSetStatistics function